commit 859f414e87163318b9d3616e9105b7f003647ca6 (refs/changes/50/1150/1)
Author: Bryan Cantrill <bryan@joyent.com>
Date:   2016-12-20T15:24:22-08:00 (2 years, 10 months ago)
    
    joyent/manta-thoth#145 modernize paths for sdc-thoth and sdc-thoth-install

diff --git a/bin/sdc-thoth b/bin/sdc-thoth
index 3ef93e6..1d371af 100755
--- a/bin/sdc-thoth
+++ b/bin/sdc-thoth
@@ -1,7 +1,7 @@
 #!/usr/node/bin/node
 
 /*
- * Copyright (c) 2013, Joyent Inc. All rights reserved.
+ * Copyright (c) 2016, Joyent Inc. All rights reserved.
  */
 
 mod_amqp = require('/smartdc/node_modules/amqp');
@@ -24,8 +24,8 @@ var exchange;
 var config;
 var connection;
 var writable = true;
-var path = '/usr/node/bin:/opt/custom/thoth/node_modules/thoth/bin:' +
-    process.env.PATH;
+var path = '/usr/node/bin:/opt/custom/thoth/node_modules/manta-thoth/bin:' +
+    '/opt/custom/thoth/node_modules/thoth/bin:' + process.env.PATH;
 var socket;
 
 process.env.PATH = path;
diff --git a/bin/sdc-thoth-install b/bin/sdc-thoth-install
index a7d6326..3bddbcd 100755
--- a/bin/sdc-thoth-install
+++ b/bin/sdc-thoth-install
@@ -115,8 +115,8 @@ if [ ! -d $base ]; then
 fi
 
 PATH=$base/bin:$PATH
-PATH=$base/node_modules/thoth/bin:$PATH
-PATH=$base/node_modules/thoth/node_modules/manta/bin:$PATH
+PATH=$base/node_modules/manta-thoth/bin:$PATH
+PATH=$base/node_modules/manta-thoth/node_modules/manta/bin:$PATH
 PATH=$base/node_modules/smartdc/bin:$PATH
 export PATH
 
